invenio_records_ui/permissions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
@lnielsen @jirikuncar 
what would you like to do with the [docs](https://github.com/inveniosoftware/invenio-records-ui/blob/master/invenio_records_ui/__init__.py#L250-L284)?
Could you help me to change it (if needed)?